SqsSourceMockSpec: SqsSource should enable throttling on emptyReceives and disable throttling when a new message arrives

https://travis-ci.com/github/akka/alpakka/jobs/380666705
```
[info] - should enable throttling on emptyReceives and disable throttling when a new message arrives *** FAILED *** (62 milliseconds)
[info]   java.lang.AssertionError: Expected OnNext(_), yet no element signaled during 10 milliseconds
[info]   at akka.stream.testkit.TestSubscriber$ManualProbe.expectNext(StreamTestKit.scala:347)
[info]   at akka.stream.testkit.TestSubscriber$Probe.requestNext(StreamTestKit.scala:802)
[info]   at akka.stream.alpakka.sqs.scaladsl.SqsSourceMockSpec.$anonfun$new$10(SqsSourceMockSpec.scala:144)
[info]   at akka.stream.alpakka.sqs.scaladsl.SqsSourceMockSpec.$anonfun$new$10$adapted(SqsSourceMockSpec.scala:144)
[info]   at scala.collection.immutable.Range.foreach(Range.scala:158)
[info]   at akka.stream.alpakka.sqs.scaladsl.SqsSourceMockSpec.$anonfun$new$9(SqsSourceMockSpec.scala:144)
[info]   at akka.stream.testkit.scaladsl.StreamTestKit$.assertAllStagesStopped(StreamTestKit.scala:32)
[info]   at akka.stream.alpakka.sqs.scaladsl.SqsSourceMockSpec.$anonfun$new$8(SqsSourceMockSpec.scala:101)
```

--- a/sqs/src/test/scala/akka/stream/alpakka/sqs/scaladsl/SqsSourceMockSpec.scala
+++ b/sqs/src/test/scala/akka/stream/alpakka/sqs/scaladsl/SqsSourceMockSpec.scala
@@ -93,7 +93,7 @@ class SqsSourceMockSpec extends AnyFlatSpec with Matchers with DefaultTestContex
       i <- 1 to bufferToBatchRatio
       message <- defaultMessages
     } {
-      probe.requestNext(10.milliseconds) shouldEqual message
+      probe.requestNext() shouldEqual message
     }
     probe.cancel()
   }
@@ -141,7 +141,7 @@ class SqsSourceMockSpec extends AnyFlatSpec with Matchers with DefaultTestContex
         .withWaitTime(timeout)
     ).runWith(TestSink.probe[Message])
 
-    (1 to firstWithDataCount * 10).foreach(_ => probe.requestNext(10.milliseconds))
+    (1 to firstWithDataCount * 10).foreach(_ => probe.requestNext())
 
     verify(sqsClient, atMostTimes(firstWithDataCount + parallelism)).receiveMessage(any[ReceiveMessageRequest])
 

